packages/Telephony/Readme.md

```diff
@@ -13,18 +13,17 @@ Like any other channel, Telephony channel allows you to transfer call to an agen
 * PhoneNumber
 
 #### Usage
-* Phone Number is not empty and in the E.164 format. 
+* Phone Number should not be empty and should be in the E.164 format. 
 * The call transfer action is only valid when called in a conversation on the Telephony channel. The action can be considered a No-op for all other channels.
 
 #### Dialog Flow
 * Once the call transfer is completed, the bot is removed from the current conversation and control is transferred to the extenal phone nunber.
-* [Open] Do we get back a handoff status on success? Should we wait for the status in our package?
+* The bot will not get any handoff status on success.
```

Comment:
The current behavior is 
1. Handoff Success : gets End of Conversation and the call ends
2. Handoff failure: gets a handoff status with a failed message.

Reply:
yes added in failure flow